packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -630,6 +630,16 @@ const schema: SchemaType<DbTag> = {
     group: formGroups.advancedOptions,
     nullable: true,
   },
+  noindex: {
```

Comment:
`noindex` and `noIndex` are used about an equal amount in the codebase at the moment, I went with `noindex` for no particular reason